plugin/storage/es/spanstore/service_operation_test.go

```diff
@@ -31,9 +31,15 @@ func TestWriteService(t *testing.T) {
 		indexService := &mocks.IndexService{}
 
 		indexName := "jaeger-1995-04-21"
+		service := Service{
+			ServiceName:   "service",
+			OperationName: "operation",
+		}
+		serviceHash := service.hashCode()
+
 		indexService.On("Index", stringMatcher(indexName)).Return(indexService)
 		indexService.On("Type", stringMatcher(serviceType)).Return(indexService)
-		indexService.On("Id", stringMatcher("service|operation")).Return(indexService)
+		indexService.On("Id", stringMatcher(serviceHash)).Return(indexService)
```

Comment:
I'm not a big fan of using the same hash function here to test that the service was hashed, I'd prefer if you generated this hash once and compare against the actual hash string.

Reply:
I've removed calls to `hashCode()` and put static hash in those tests.